src/lib/scanner.ts

```diff
@@ -137,77 +132,25 @@ export class ScanRunner {
         });
     }
 
-    /**
-     *
-     * @param executionResult The results from a scan
-     * @returns The HTML-formatted scan results, or an empty string.
-     * @throws If {@code executionResult.result} is not a string.
-     * @throws If {@code executionResult.warnings} contains any warnings about methods not being found.
-     * @throws if {@code executionResult.status} is non-zero.
+        /**
+     * Uses the provided arguments to run a Salesforce Code Analyzer command.
+     * @param args The arguments to be supplied
      */
-    private processDfaResults(executionResult: ExecutionResult): string {
```

Comment:
I'm surprised that we can remove this method. I would have thought that we would be processing the same text (whether it was retrieved from the display or from json output) just as before.

Reply:
I need @jfeingold35's inputs here. But I think the executionResult coming out of CLI stdout had other parameters and we needed this to parse and just get to the HTML. Based on my testing, by directly calling the runDfaAction, we only get the plain HTML and I don't see any issues with simply casting than to a string and showing it to the users. Let me know if I'm missing anything.

Reply:
@stephen-carter-at-sf - Checked on the warning and error cases and we are good. The only case that was being handled was related to "no method found" and there isn't a possibility of this occurring via VS Code. I ran through this scenario with @jfeingold35 offline and he agrees on this approach.